vimrc

```diff
@@ -106,8 +106,18 @@ inoremap <S-Tab> <c-n>
 " Exclude Javascript files in :Rtags via rails.vim due to warnings when parsing
 let g:Tlist_Ctags_Cmd="ctags --exclude='*.js'"
 
+
 " Index ctags from any project, including those outside Rails
-map <Leader>ct :!ctags -R .<CR>
+function! ReindexCtags()
+  let l:ctags_hook_location = '$(git rev-parse --show-toplevel)/.git/hooks/ctags'
+
+  if exists(l:ctags_hook_location)
+    exec '!'. l:ctags_hook_location
+  else
+    exec "!ctags -R ."
+  endif
+endfunction
+nmap <Leader>ct :call ReindexCtags()<CR>
```

Comment:
I'm definitely interested in going in this direction. Are we going to place those files in `.vim/configs` (so this would go in `.vim/configs/ctags.vim`) and then source those files? I'm worried about placing them at the root level, but also want to cause the least possible disruption to anyone's existing configuration.